### PR TITLE
tests: move test scripts to separate directory

### PR DESCRIPTION
**Pull request by williamcroberts**
Organize the test scripts into a supordinate directory.

This makes finding tests easier as it doesn't really rely on
magic name prefixes.

Signed-off-by: William Roberts <william.c.roberts@intel.com>

### CODE REVIEW

File: tools/tpm2_getmanufec.c [hunk 1]
```diff
@@ -410,15 +410,19 @@ int RetrieveEndorsementCredentials(char *b64h)
...
+        goto out;
     }
 
     LOG_INFO("%s", b64);
 
-    int rc = RetrieveEndorsementCredentials(b64);
+    rc = RetrieveEndorsementCredentials(b64);
+out:
+    free(hash);
     free(b64);
```
Comment (martinezjavier) [inline tools/tpm2_getmanufec.c]: This is an unrelated change and should be in a separate patch.

Comment (williamcroberts) [reply tools/tpm2_getmanufec.c]: It is, it's on 0ec3a5a where as the move is on ff44794

Comment (martinezjavier) [reply tools/tpm2_getmanufec.c]: @williamcroberts ah, sorry. For some reasons I thought it was part of the same commit. Sorry for the noise.
